crates/version/src/lib.rs

```diff
@@ -1,58 +1,49 @@
+use std::borrow::Cow;
 use std::fmt;
 use std::sync::LazyLock;
 
 #[cfg(test)]
 mod tests;
 
-pub struct CalimeroVersion {
-    release: &'static str,
-    build: &'static str,
-    commit: &'static str,
-    rustc: &'static str,
-    protocol: &'static str,
+static CURRENT: LazyLock<CalimeroVersion<'static>> = LazyLock::new(|| CalimeroVersion {
+    release: Cow::Borrowed(env!("CARGO_PKG_VERSION")),
+    build: Cow::Borrowed(env!("CALIMERO_BUILD")),
+    commit: Cow::Borrowed(env!("CALIMERO_COMMIT")),
+    rustc: Cow::Borrowed(env!("CALIMERO_RUSTC_VERSION")),
```

Comment:
this should work too, less clutter

```suggestion
    release: env!("CARGO_PKG_VERSION").into(),
    build: env!("CALIMERO_BUILD").into(),
    commit: env!("CALIMERO_COMMIT").into(),
    rustc: env!("CALIMERO_RUSTC_VERSION").into(),
```